framework/resources/Functional/filter/pushdown/filter_pruning/data.json

```diff
@@ -0,0 +1,27 @@
+{
+  "testId": "filter_pushdown_filter_pruning",
+  "type": "group",
+  "description": "Filter pushdown tests: these tests verify the query result where the filter pruning is taking place",
+  "categories": [
+    "functional"
+  ],
+  "matrices": [
+    {
+      "query-file": ".*.sql",
+      "schema": "dfs.drillTestDir",
+      "output-format": "tsv",
+      "expected-file": ".*.e_tsv",
+      "verification-type": [
+        "in-memory"
+      ]
+    }
+  ],
+  "datasources": [
+    {
+      "mode": "cp",
+      "src": "Datasources/parquet_storage/DRILL_3855/DRILL_3855_test_data",
+      "dest": "parquet_storage/DRILL_3855/DRILL_3855_test_data"
```

Comment:
do you think this can be simpifiled - like remove this sub-dir or rename it as just data, as we know it's under DRILL-3855?

Reply:
Your'e right. Removed the redundant sub-dir here and in other filter pushdown tests as well. I have a concern though: as the data is already on clusters there would be duplicates. Don't sure about the best way to solve this except to clean it manually.

Reply:
Actually, I was talking about the sub dir. 

It’ll be good to have </drill/testdata>/<feature>/<JIRA>/<data>

When you do a hadoop ls, it’ll show consistent directories for JIRAs. Some may have one file and some may have many files.

Reply:
Moved the data to the </drill/testdata>/filter/pushdown folder. Most of it are Drill tables, which themselves represented as folders, so we don't need any intermediate directory. And so I left JIRA name as  part of the table names.